components/molecule/accordion/src/AccordionItemPanel.js

```diff
@@ -41,10 +41,9 @@ const AccordionItemPanel = forwardRef(
       <div
         id={id}
         ref={forwardedRef}
-        role="region"
         className={cx(BASE_CLASS_ITEM_PANEL, BASE_CLASS_ELEMENT)}
         aria-expanded={values.includes(value)}
-        aria-labelledby={headerId}
+        data-testid="accordion-panel"
```

Comment:
I’m not sure if adding this test attribute in production is the best solution.
Couldn’t we expose the component’s props and pass this attribute only from the test?

Reply:
Prop spreading.